snmp/datadog_checks/snmp/data/default_profiles/cyberpower-pdu.yaml

```diff
@@ -0,0 +1,373 @@
+extends:
+  - _base.yaml
+
+sysobjectid: 1.3.6.1.4.1.3808.1.1.*
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "cyberpower"
+
+metrics:
+  - MIB: CPS-MIB
+    table:
+      OID: 1.3.6.1.4.1.3808.1.1.3.2.3.1
+      name: ePDULoadStatusTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3808.1.1.3.2.3.1.1.2
+        name: ePDULoadStatusLoad
```

Comment:
I think we need a prefix for this profile since `ePDU` prefix is not specific enough.

A possible prefix is  `cyberpower`, we will have metrics linke  `snmp.cyberpower.ePDULoadStatusLoad`

Alternative is `cps` prefix. I think CPS means cyberpower systems.

Reply:
done